src/app/components/Billboard/BillboardCurationGrid/index.tsx

```diff
@@ -1,35 +1,59 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/react';
+import moment from 'moment';
 import styles from './index.styles';
 import CurationPromo from '../../Curation/CurationPromo';
 import { CurationGridProps } from '../../Curation/types';
 
 const BillboardCurationGrid = ({
   summaries,
-  eventTrackingData: _eventTrackingData, // Accept for use in other PR
+  eventTrackingData,
 }: CurationGridProps) => {
   const top4SummariesForGrid = summaries.slice(0, 4);
 
-  if (top4SummariesForGrid.length === 0) {
-    return null;
-  }
+  if (top4SummariesForGrid.length === 0) return null;
 
   const hasMultiplePromos = top4SummariesForGrid.length > 1;
   const firstPromo = top4SummariesForGrid[0];
 
+  const withItemTracking = (promo: typeof firstPromo, index: number) => ({
+    ...eventTrackingData,
+    itemTracker: {
+      ...(eventTrackingData.itemTracker || {}),
```

Comment:
Don't think we need to spread `itemTracker` here as it'll be empty at this point.